extras/profile-and-settings.rst

```diff
@@ -73,6 +73,10 @@ Clicking on "Profile" in the user menu opens the profile settings.
    has no effect. Most common modern Browsers are capable to do so
    (e.g. Firefox, Google Chrome).
 
+   Additionally, you can activate or deactivate Zammad's ticket summary
+   :doc:`AI feature <ai-features>`. This option
+   is only available, if the feature is configured by your admin.
+
```

Comment:
Perhaps a misunderstanding, but this confused me: the checkbox controls the visibility of the banner only, not the actual feature! The sidebar is always accessible for an agent when it is configured in the system.